src/main/java/com/ecwid/maleorang/method/v3_0/reports/email/EmailReportActivityDetails.kt

```diff
@@ -28,5 +28,14 @@ class EmailReportActivityDetails : MailchimpObject() {
     @JvmField
     @Field
     var ip: String? = null
-	
+
+
+    fun setAction(action: String) {
```

Comment:
I don't think it is a good idea to have such methods, and they are not used as I see.